CCDB/src/CcdbApi.cxx

```diff
@@ -27,6 +27,8 @@
 #include <TWebFile.h>
 #include <TH1F.h>
 #include <TTree.h>
+#include <FairLogger.h>
+#include <TError.h>
```

Comment:
Why do you need TError? (FairLogger defines its own ERROR macro I think).

Reply:
I need it to decrease the verbosity of ROOT itself during the reading of the TFile.

Reply:
That's what I figured out in between. So it's already merged!